src/Makefile.am

```diff
@@ -1966,11 +1966,11 @@ tests_testHttpRange_LDADD = \
 	acl/libapi.la \
```

Comment:
Please reorder squid_LDADD as well to keep the order of the two libraries consistent. Currently, squid_LDADD still uses the opposite order:

```am
squid_LDADD = \
...
        ip/libip.la \
...
        comm/libcomm.la \
```

I do not insist on this change. I am aware that you may not be able to test on Solaris.